### ISSUE TITLE
[BUG] Problems on RG28xx

### ISSUE THREAD

**Issue by BlueJadeite** [loki666/ROCKNIX#12]
### Describe the bug
Hello, I am a rg28xx user. I found problems with 20241214.

1. Unable to control with File Manager
The controller is not responding after rc3.

2. LAN and WiFi are not connected to the internet
I have a LAN to USB cable and a WiFi dongle, both work with muOS.
But neither work with 20241214.

3. When I connect DC, the device turns on by itself

I am enjoying using this program. Thank you.

### Information
 - ROCKNIX Version: 20241214
 - Hardware Platform: RG28xx

**Comment by loki666** [comment]
I'll look into 1.
2 and 3 are out of reach with the current mainline kernel and u-boot state.

**Comment by BlueJadeite** [comment]
I understand that it will take time to solve problems 2 and 3.
I would be happy if we could just solve problem 1.

**Comment by loki666** [comment]
@BlueJadeite can you test this build https://github.com/loki666/ROCKNIX/releases/tag/rc4
check if it works in filemanager, portmaster, and ppsspp, duckstation-sa, mupen64-sa
thanks

do clean image install, as update might leaves some old config files

**Comment by BlueJadeite** [comment]
I tried to check rc4 with clean install.

File Manager: Controllable, but down scroll is unable. It needs push each time. Up scroll is no problem.
Portmaster: Sometimes controllable. The down button gets held frequently.
PPSSPP: Controllable.
Retroarch 32 and 64: Uncontrollable with rc4. I confirmed that it worked with rc2.
Duckstation-SA: Controllable.
Mupen64-SA: Unable to into menu. Uncontrollable in game because the analog stick is not assigned. RG28xx has no analog stick.

Thank you.
